### PR TITLE
Adding package plasmaboundaries

### PR DESCRIPTION
**Pull request by RemiTheWarrior**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by RemiTheWarrior** [comment]
@Shimwell

File: recipes/plasmaboundaries/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  license_file: LICENSE.txt
+  summary: 'Determine plasma flux functions for various plasma parameters and plasma configurations.'
+
+  description: |
+    'This code computes and plots analytical solutions of the Grad-Shafranov (GS) equation for studying plasma equilibrium,
+    stability and transport in fusion reactors based on the work of A. Cerfon and J. Freidberg. Details on the method can be
+    found in "One size fits all" analytical solutions to the Grad-Shafranov equation, Physics of Plasmas 17 (2010).'
+
+  doc_url: https://plasma-boundaries.readthedocs.io/en/latest/
+  dev_url: https://https://github.com/RemiTheWarrior/plasma-boundaries
```
Comment (dopplershift) [inline recipes/plasmaboundaries/meta.yaml:46]: Bad URL.

File: recipes/plasmaboundaries/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - scipy
+    - numpy
+    - sympy
+
+test:
+  imports:
+    - plasmaboundaries
+
+about:
+  home: https://https://github.com/RemiTheWarrior/plasma-boundaries
```
Comment (dopplershift) [inline recipes/plasmaboundaries/meta.yaml:34]: Bad URL here too

Comment (RemiTheWarrior) [reply recipes/plasmaboundaries/meta.yaml]: Fixed

File: recipes/plasmaboundaries/meta.yaml (hunk 1 above)
Comment (RemiTheWarrior) [reply recipes/plasmaboundaries/meta.yaml]: Fixed

**Review by dopplershift — APPROVED** [review]

**Comment by Shimwell** [comment]
Perfect, thanks for adding this Remi and Dopplershift. looking forward to using it

**Pull request merged**